Observation/Observation-NeoplasticCellKayBurbridge-Example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-NeoplasticCellKayBurbridge-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "55342001",
+        "display": "Neoplastic growth"
```

Comment:
switch this code for text as Neoplastic growth is a disorder, where as we are looking for an observable entity **Neoplastic cell content** (which doesn't exist in SNOMED)